src/main/java/firebase/sdk/dashboard/dao/UserDaoDatastore.java

```diff
@@ -0,0 +1,87 @@
+package firebase.sdk.dashboard.dao;
+
```

Comment:
Looks like these changes are from the other PR for UserDao implementation. Not sure why they are here, though you are diffing this against master. Please get rid of these.